### DISCUSSION TITLE
Getting incoming chat messages in notification instead of "sent a message"

### DISCUSSION THREAD

**Discussion by RevealedSoulEven** [rodit/SnapMod#125] [Ideas & Feature Suggestions]
As you know snapchat sends notification for chats but it's written "sent a message" which is quite boring, instead if the content of the message is displayed as you already added chat preview option, so please add a function to let us see the incoming messages in the notification.

**Comment by rodit** [comment]
From my memory, when I first looked at doing this, the incoming message content typically isn't available at the time of receiving the notification. I'll check again but I don't think this is possible in general unfortunately (it might work sometimes). 

I do really like the idea of this feature which is why I already looked at it before, so hopefully I can figure it out.

**Reply by RevealedSoulEven** [reply]
Aahhh if that's the case then maybe it won't come true.
But yeaa... Hope to see something from you soon regarding it. Thanks btw!

**Reply by rodit** [reply]
I did, by some miracle, figure out how to do this - you'll see it in the next update.

**Comment by rodit** [comment]
Added in 1.7.0

**Reply by RevealedSoulEven** [reply]
Waiting for it❤️

**Reply by RevealedSoulEven** [reply]
Yeaa it's there. Thanks btw❤️ And great job

**Reply by viridivn** [reply]
This is wonderful, you have my infinite gratitude for this mod!
